src/components/Categories.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Categories() {
+  return (
+    <div>
+      Under construction
+    </div>
+  );
+}
```

Comment:
- [ ] As in the project requirements the component should contain:
  - A button for checking the status and this button would use a function `checkCategoriesStatus` (name can be different) made in the component (name may be different)
    - `checkCategoriesStatus`  dispatches the` underConstruction` action creator (the name of the function can be different)

Please kindly fix that 🙃 😉